documentation/VerifiableCredentials.md

```diff
@@ -1,5 +1,7 @@
 ## Running the Aries Agent Locally for Verifiable Credential Testing and Development
 
+**Please Note** The following process is only necessary when working with Verifiable Credential Testing and Development, ie not required when initially setting up the project. 
```

Comment:
Looks great!!!  

I would probably use `**NOTE:**`, and replace `ie` with `, and is not necessary for general development purposes.  For example, when initially setting up the project.`

Reply:
Can we eliminate the sentence fragment, i.e. `... general development purposes, for example, when ...`

Reply:
If you want to eliminate the sentence fragment it would be `purposes; for example, when initially`